lib/report/timeline.go

```diff
@@ -38,6 +39,8 @@ func (r TimelineCollector) Collect(ctx context.Context, reportWriter FileWriter,
 		return trace.Wrap(err)
 	}
 	defer w.Close()
+	ctx, cancel := context.WithTimeout(ctx, defaults.StatusCollectionTimeout)
```

Comment:
During testing, I've been hitting the issues with abort when the planet container has not yet initialized properly (and hence the agent was not running). It does not really make sense to wait for the agent indefinitely (or for the lifetime of the parent context which defaults to 10min).